### ISSUE TITLE
QGEP in french or german

### ISSUE THREAD

**Issue by ponceta** [QGEP/QGEP#657]
As I've seen from the documentation, it's not a simple operation for a user to get a translated project in french or german.

We should provide it if we can, at least basically, even if it's not fully translated.

Having this option would push contributions towards the translations.

**Comment by sjib** [comment]
@ponceta Do you think it is too complicated just to leave the qm file in the same folder? 
[1.4.5. Install the demo project](https://qgep.github.io/docs/installation-guide/workstation.html#install-the-demo-project) 

What then could be improved?
Or what do you mean with a translated project? Translated data or just translated user interface?

**Comment by ponceta** [comment]
@sjib That's maybe because I didn't find where to look for the qgep_fr.qm file :S

And I won't be the only one I think.

**Comment by sjib** [comment]
@ponceta I tried to document this better - what do you think?

**Comment by ponceta** [comment]
What I missed was actually the qgep.zip file in the latest release. :(

* Download from https://github.com/QGEP/qgep/releases/latest the qgep.zip

qm language files are not tracked by git and therefore it is complicated for the user to understand why they are not in the folder when he clones the project.

How can we gather / refresh these qm files? Would it be possible to keep these tracked by git? I assume they work with specific versions of the datamodel, it would be nice to update these in git and then add these in the release process?

**Comment by m-kuhn** [comment]
They are created from the qgis project and the transifex web service. They are [produced as part of the build/release process](https://github.com/QGEP/QGEP/blob/master/.github/workflows/create_translated_project.yaml#L18-L19) with

```
pip install transifex-client
./scripts/pull_ts_and_create_project.sh
```

Adding them to the git repository would increase complexity / raise maintenance responsibility questions.

They are not directly connected to the data model.

**Comment by ponceta** [comment]
@m-kuhn Thank you for this feedback!

+1 for having it automated for releases.

**Comment by m-kuhn** [comment]
> +1 for having it automated for releases.

That is the case now.
Is this issue done or are there things left to do in here (better docs?)

**Comment by ponceta** [comment]
First merge the doc PR, this issue will close automatically ;)

**Comment by ponceta** [comment]
![image](https://user-images.githubusercontent.com/8614223/127104661-a0640d5e-b035-4b76-a1ed-415663e03ce9.png)

I post this here as a reminder

**Comment by m-kuhn** [comment]
I wasn't aware of that.
Left a feedback, ready to merge now.

> First merge the doc PR, this issue will close automatically ;)

**Comment by ponceta** [comment]
@m-kuhn This didn't work because wrong keyword (@sjib says Should solve QGEP/QGEP#657 instead of FIX QGEP/QGEP#657)

List of keywords and more about that :

https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
